### ISSUE TITLE
Errors with protected property

### ISSUE THREAD

**Issue by rtilte-reol** [merorafael/yii2-monolog#3]
I installed the 0.1.x version today and tried the wrapper as explained. The following Exception occurs:

```
Exception 'yii\base\UnknownPropertyException' with message 'Setting unknown property: Mero\Monolog\MonologComponent::channels'
```

As for all components in Yii2, the channels needs to be declared as public (or maybe I'm missing something somewhere). The other solution would be to add a setter for this property.

Also, I saw that this has been modified on the master branch, but `$handlerConfig['formatter']` may not exist, which triggers an error when testing it in MonologComponent #56.

**Comment by merorafael** [comment]
Thank's for reporting @rtilte-reol. I hope to soon finish the 0.2.0 version, which is covered by unit tests and such unexpected behavior should not happen.

The bug as been fix and soon I will create a 0.1.2 release.

**Comment by merorafael** [comment]
Update 0.1.2 released with fixes.
